cpp/util/fake_etcd.cc

```diff
@@ -25,6 +25,35 @@ using util::StatusOr;
 using util::Task;
 
 namespace cert_trans {
+namespace {
+
+
+string NormalizeKey(const string& input) {
```

Comment:
This doesn't really seem to normalise per se, just collapse repeated slashes (doesn't handle `..` for example. What does etcd do with things like that?)